test/unit/presenters/business/business-details-presenter.test.js

```diff
@@ -153,14 +154,89 @@ describe('businessDetailsPresenter', () => {
   })
 
   describe('the "cph" property', () => {
-    describe('when the property is empty', () => {
-      test('it should return empty array', () => {
+    describe('when there are multiple CPH numbers', () => {
```

Comment:
We could have added a test case or edited an existing one to cover the changes but all good, non blocking